guides/release/components/passing-arguments-and-html-attributes.md

```diff
@@ -0,0 +1,159 @@
+There are two kinds of things that you can pass to a component when you use it in a template,
+named arguments, and HTML attributes.
+
+If we have the following template:
+
+```handlebars
+<MyComponent @arg="Argument" class="error">
+```
+
+We can say that we are passing the `arg` named argument
+and the `class` HTML property to the `MyComponent` component.
+
+The way to distinguish them is that named arguments are prefixed with `@`,
+while HTML attributes are not.
+
+## Named arguments
+
+A named argument is a value passed to the component when invoking it.
+
+We want to greet our friend by passing their name to the component.
+The JavaScript file does not need to change,
+but we will update the template to reference `friend`:
+
+```javascript
+import Component from '@glimmer/component';
+
+export default class Greeting extends Component {
+  constructor() {
+    super(...arguments);
+
+    this.name = "Tomster";
+  }
+}
+```
+
+```handlebars
+<p>Hello {{@friend}}, I'm {{this.name}}!</p>
+```
+
+Now when we call the `Greeting` component like so:
+
+```handlebars
+<Greeting @friend="Zoey" />
+```
+
+The output will be:
+
+```html
+<p>Hello Zoey, I'm Tomster!</p>
+```
+
+In this example, `{{@friend}}` is referring to a named argument,
+a value that is given to the component when it is invoked.
+`{{this.name}}` is referring to a property of the component.
+
+This distinction is useful when reading template files.
+If you see `{{@}}`, you know the value comes from outside of the component.
+If you see `{{this.}}`, you know the value comes from the component's JavaScript object.
```

Comment:
this was already covered in the "displaying data" section. We may want to tighten that explanation up, and just refer to it from here.